src/components/Article.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../App.scss';
+
+function Article(props) {
```

Comment:
```suggestion
function Article({ title, date, text }) {
```
Используй деструктуризацию